demo-app/src/open-truss/components/ClientPageTitle.tsx

```diff
@@ -1,8 +1,24 @@
 'use client'
-import { type BaseOpenTrussComponentV1Props } from '@open-truss/open-truss'
+import { z } from 'zod'
+import { BaseOpenTrussComponentV1PropsShape } from '@open-truss/open-truss'
 
-function PageTitle({ children }: BaseOpenTrussComponentV1Props): JSX.Element {
-  return <h1>{children}</h1>
+export const Props = BaseOpenTrussComponentV1PropsShape.extend({
```

Comment:
This exporting of `Props`, a zod shape of what props are available, is what powers the config builder. We use `describeZod` to translate this into something that the config builder uses to create inputs for choosing what props to use.